README.md

```diff
@@ -256,22 +267,91 @@ kpt alpha repo register \
+```
+
 ## Installing Config Sync in Workload Clusters
 
-Workload clusters - those clusters that do not contain the Nephio system itself,
-but instead are intended to run the workloads deployed via Nephio - must run
-Config Sync to get their workloads from their deployment repositories.
+Workload clusters must run Config Sync to get their workloads from their deployment repositories.
 
 The package
-[nephio-configsync](https://github.com/nephio-project/nephio-packages.git/nephio-configsync)
+[nephio-configsync](https://github.com/nephio-project/nephio-packages/tree/main/nephio-configsync)
 is intended for installing Config Sync in those clusters. To use this package,
 you will need to update the RootSync resource with the repository and
 authentication needed for your environment.
 
 Example:
 ```
-kpt pkg get --for-deployment https://github.com/nephio-project/nephio-packages.git/nephio-configsync test-deploy-edge-01
-kpt fn render test-deploy-edge-01
-kpt live init test-deploy-edge-01
-kpt live --context=gke_jbelamaric-dev_us-central1_nephio-edge-01 apply test-deploy-edge-01 --reconcile-timeout=15m --output=table
+kpt pkg get --for-deployment https://github.com/nephio-project/nephio-packages.git/nephio-configsync nephio-test-deploy-01
+# Replace the repository name in rootsync.yaml with the $GITHUB_USERNAME
+kpt fn eval nephio-test-deploy-01 \
+  --save \
+  --type mutator \
+  --image gcr.io/kpt-fn/search-replace:v0.2.0 \
+  -- by-path=spec.git.repo by-value-regex='https://github.com/[a-zA-Z0-9-]+/(.*)' \
+  put-value="https://github.com/${GITHUB_USERNAME}/\${1}"
+kpt fn render nephio-test-deploy-01
+kpt live init nephio-test-deploy-01
+kpt live apply nephio-test-deploy-01 --reconcile-timeout=15m --output=table
+
+# Verify that the config sync pods are running
+kubectl get pods --all-namespaces | grep config
+config-management-monitoring   otel-collector-5b8d5d8d8f-8qs2k                                 1/1     Running   0          1m
+config-management-system       config-management-operator-6d7867f9f5-xb9cn                     1/1     Running   0          3m
+config-management-system       reconciler-manager-c6c8cf7f6-9x7vr                              2/2     Running   0          1m
+config-management-system       root-reconciler-nephio-workload-cluster-sync-679dd89788-mkl8x   4/4     Running   0          1m
+```
+
+## Deploying a Package Workload
+We have two options to provision a sample workload.
+  * Use the Nephio Web UI in case you followed the steps in [Installing the Nephio Web UI](#installing-the-web-ui) above, or
+  * Use `kpt` and `git` to provision the workload from the command line.
+
+[Installing the Nephio Web UI](#installing-the-web-ui) (as described above) was optional.
+Therefore, we will use `kpt` and `git` to provision the workload.
+
+Example:
+```
+git clone https://github.com/${GITHUB_USERNAME}/nephio-test-deploy-01.git caching-dns
```

Comment:
You can just clone the whole repo, no need to name the local clone "caching-dns". Once you clone the repo, as you pull additional packages in, you can just put them in there same as you did this one.

Reply:
Got it. Makes sense.
Based on your other comment below I have removed the git option as I like the pure kpt approach you mentioned way better.